docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,257 @@
+# NUI Widget Application
+
+[NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html)  is to show a small version of application on home screen.
+
+**Figure: NUI widget application**
+
+![NUI widget application](./media/dali_widget_application.png)
+
+<br>
+The main widget application features include:
+
+- Creating widget applications
+
+  You can [Create a NUI Widget Application](#create-a-nui-widget-application)
+, which usually has 1 process for maintaining the main loop. Within the process, the framework can be create widget instance that can share the same resources.
+  The Widget application can also share data with other applications.
+
+- Managing widget instances
+  
+  Whenever a widget instance is requested, the framework creates one.
+  You can manage the widget instances by updating or destroying them, or by retrieving information about them.
+
+- Managing the life-cycle
+  
+  You can manage the widget instance life-cycle through callback methods that are triggered as the instance state changes.
+
+- Creating the widget UI
+
+  The widget application can draw a UI on the Home screen or viewer
+
+
+    > **Note**
+    >
+    > The widget application UI has a limitation with the scroll action to provide a better user experience. Design the widget UI to display all the information within the given area of the screen points.
+    >
+    > To draw the UI, you must use the [Window](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.Window.html) received from the OnCreate Callback.  Do not create additional windows. A stack of widget application windows gets corrupted, because the platform handles the widget application window in a special way.
```

Comment:
Suggest rephrasing to:

> To draw the layout, you must use the [Window](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.Window.html) received from the `OnCreate` callback.  If you create additional windows, the stack of widget application windows gets corrupted. This is because the platform handles the widget application window in a particular way.